src/Backend/Core/Js/backend.js

```diff
@@ -364,10 +364,10 @@ jsBackend.ckeditor = {
     uploadUrl: '/src/Backend/Core/Js/ckfinder/core/connector/php/connector.php?command=QuickUpload&type=Files&responseType=json',
 
     // load some extra plugins
-    extraPlugins: 'stylesheetparser,mediaembed',
```

Comment:
Is this changed by intention or accidentally?

Reply:
intentionally, we didn't use the mediaembed and the others needed to be activated because the build is slightly different

Reply:
@jeroendesloovere do you need any other changes to this pr?